### ISSUE TITLE
Multiple approaches for loading cache data from print page

### ISSUE THREAD

**Issue by FriedrichFroebel** [tomasbedrich/pycaching#136]
There currently are two methods which load cache data from the print page: `Cache._from_print_page` (a class method) and `Cache.load_by_guid` (a regular method). The first one uses a language-independent approach, while the second one relies on an English page. In the second case there even is a to-do comment to not rely on an English page.

For this reason, we should probably try to reduce the duplicates there and switch to the language-independent approach.

**Comment by tomasbedrich** [comment]
Good idea! The two versions somehow sneaked up into the code over the years.

**Comment by FriedrichFroebel** [comment]
Proposal: Move the code from `_from_print_page` to `load_by_guid`. Then deprecate `_from_print_page` (it has been marked as internal anyway) and basically redirect to `load_by_guid` for the deprecation period.
